src/widgets/programs/programs.resource.tsx

```diff
@@ -18,6 +18,20 @@ export function fetchEnrolledPrograms(
   );
 }
 
+export function fetchActiveEnrollments(
+  patientID: string
+): Observable<PatientProgram[]> {
+  return openmrsObservableFetch(
+    `/ws/rest/v1/programenrollment?patient=${patientID}`
+  ).pipe(
+    map(({ data }) => {
+      return data["results"]
```

Comment:
Why do we need to access it like this? Can we provide proper typings for the response - that we `data.results` is valid and type checked.

Reply:
I've tried to address this in the latest commit.